tests/kafkatest/services/verifiable_producer.py

```diff
@@ -55,16 +79,32 @@ def _worker(self, idx, node):
                     elif data["name"] == "producer_send_success":
                         self.acked_values.append(int(data["value"]))
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-verifiable-producer.sh" \
+    def _kafka_dir(self, node):
+        if node.version == KafkaVersion.TRUNK:
```

Comment:
Since I've seen this a couple of times now, would it make more sense to just put the trunk version under `/opt/kafka-trunk` so this method is always a one-liner: `return "/opt/kafka-" + node.version`?